cityofaustin/joplin
# 4166 import contacts

# Description
Update importer to support contacts
https://github.com/cityofaustin/techstack/issues/4166








# Types of changes



# How can this be tested?



 
* Import a service or location page with a contact:
* See the correct contact is associated with the page
  * If the contact didn't exist before importing this page, see it was created
    * If it has a location page that was already imported associated with it, see that the location page gets associated correctly (check is on slug)
  * If the contact did already exist (check is on contact name), see that it's associated correctly

# Checklist:
